File: 01_Leetcode_FAANG/Subham_Ashutosh/Shuffle_the_array.cpp
class Solution {
public:
    vector<int> shuffle(vector<int>& nums, int n) {
        vector<int> ans;
        int a=0,b=n;
        for(int i=0;i<2*n;i++)
        {
            if(i%2==0)
                ans.push_back(nums[a++]);
            else
                ans.push_back(nums[b++]);
        }
        return ans;
    }
};

File: 01_Leetcode_FAANG/Swayam Pattanaik/Shuffle_the_array.java
class Solution {
    public int[] shuffle(int[] nums, int n) {
        int j=0;
        int ans[]=new int[2*n];
        for(int i=0;i<nums.length;i++)
        {
            if(i%2==0)
            {
                ans[i] = nums[j++];
            }
            else
                ans[i] = nums[n++];
        }
        return ans;
    }
}
